### PR TITLE
Get additional information as part of data archival request template

### PR DESCRIPTION
**Pull request by balajialg**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/data_archival_request.yml [hunk 1]
```diff
@@ -8,13 +7,31 @@ body:
...
+      required: true
+  - type: input
+    attributes:
+      label: Term/Year
+      description: Identify the term of the semester (Fall/Spring/Summer) and the year of your course enrollment
+    validations:
+      required: true     
+  - type: input
+    attributes:
+      label: Login Detail
```
Comment (yuvipanda) [inline .github/ISSUE_TEMPLATE/data_archival_request.yml:31]: can i ask why we are asking this? The identifier provided in the text file should be all that is needed for this, right? It also encodes the name of the hub that is archived, so it *should* say which hub it is.

**Review by yuvipanda — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/data_archival_request.yml (hunk 1 above)
Comment (balajialg) [reply .github/ISSUE_TEMPLATE/data_archival_request.yml:31]: @yuvipanda - My understanding is that these changes will help us have relevant data to respond to the archival request when we encounter scenarios like this - https://github.com/berkeley-dsep-infra/datahub/issues/3487. @felder Please feel free to share more details if required.

Comment (felder) [reply .github/ISSUE_TEMPLATE/data_archival_request.yml:31]: @yuvipanda The problem is a number of people open file retrieval requests without being directed to do so by the text file.  Thus far most, but not all, of those cases involve folks who have graduated.  There's also often a mix of archived and unarchived files from different hubs/filesystems for these people.  As @balajialg mentioned issue #3487 is a very good example of this issue.  This person had graduated, could no longer login, and had a mix of archived directories as well as directories I had to manually archive and send to them.

Thus I'm asking @balajialg to include language in the template that can steer folks into giving me the correct info.  

@balajialg I'd like to meet with you to discuss.  Like I don't think we should ask if people have graduated specifically.  Instead, I think we should ask something along the lines of whether or not they can still login and direct them to do so if they can.  If they cannot, I'll need information about what hub(s) they used.

Comment (yuvipanda) [reply .github/ISSUE_TEMPLATE/data_archival_request.yml:31]: thank you for pointing out to that issue @felder and @balajialg! Makes sense that is the wrong form to use for this. I'd like to keep this form simple - maybe we can have a different one for people who no longer have access? I'm not sure how this user found this form, but it feels like the problem was that they were completely in the wrong place.

**Comment by felder** [comment]
@yuvipanda when you hit "new issue" in the issue tracker, it's an option in the list.  Seems like that's the most likely way they find it.

I'm definitely open to the idea of having separate issue templates for this.  Even better if we can hide the template associated with the expected use case of being directed here via the text file.

File: .github/ISSUE_TEMPLATE/data_archival_request.yml (hunk 1 above)
Comment (felder) [reply .github/ISSUE_TEMPLATE/data_archival_request.yml:31]: @yuvipanda when you hit "new issue" in the issue tracker, it's an option in the list. Seems like that's the most likely way they find it.

I'm definitely open to the idea of having separate issue templates for this. Even better if we can hide the template associated with the expected use case of being directed here via the text file.

Comment (balajialg) [reply .github/ISSUE_TEMPLATE/data_archival_request.yml:31]: @felder Yes, find some slot in my bcal next week to discuss this more. Let's think it through from an user perspective and find an optimal solution :)

**Pull request closed without merging**